packages/preview/markly/0.2.0/README.md

```diff
@@ -0,0 +1,172 @@
+This is a [Typst](https://github.com/typst/typst) package to add cut, bleed, and registration marks to documents.
+
+It's likely that some future version of Typst will obsolete this package.  Until that time, it's nice to have **a** way to achieve this functionality with the current version.
+
+
+# Demo
+
+![Markly Demo](img.png)
+
+...can be generated with:
+
+```typst
+#import "@preview/markly:0.2.0"
+
+#let markly-context = markly.setup(
+  stock-width:4in,
+  stock-height:3in,
+
+  content-width: 3in,
+  content-height:2in,
+)
+
+#show: markly.page-setup.with(markly-context)
+
+// Here the local title template uses markly's to-bleed
+#let title(body, inset-y:12pt) = {
+  markly.to-bleed(text(white, size:2.5em, body), markly-context)
+}
+
+#title[Bannar]
+
+- Content
+```
+
+
+For a more detailed example, see [main.typ](template/main.typ) and [main.pdf](main.pdf).
+
+# Notes
+
+Since this is a bit difficult to do there are some limitations to using this package.  Here are a few examples:
+
+* `inside`/`outside` margins don't work on pages
+* The `background` field of the `page` can't be used unless there is a wrapper function so that `marks` is called also.
+
+
+# Installation
+
+Currently, this package has **not yet** been published.  To try it out, you must download it locally.  There are many ways to do this.
```

Comment:
The README will appear on Typst Universe and this might be misleading. Could you please update this section as if the package was already published?